src/apps/tabs/components/default-tab.react.js

```diff
@@ -0,0 +1,165 @@
+/**
+ * Dependencies.
+ */
+
+let React = require('react');
+let InfiniteScroll = require('react-infinite-scroll')(React);
+let moment = require('moment');
+let cache = require('lscache');
+let ProductGroup = require('./product-group.react');
+let Header = require('./header.react');
+let ProductStore = require('../../../common/stores/product');
+let api = require('../../../common/api');
+let getDay = require('../util/get-day');
+let groupByDay = require('../util/group-by-day');
+let Pane = require('../../../common/product-pane/pane.react');
+
+/**
+ * Constants.
+ */
+
+const CACHE_KEY = process.env.PRODUCTS_CACHE_KEY;
```

Comment:
In this particular case, we are using `envify`, which simply replaces all `process.env.XYZ` with the appropriate values in the environment before making the final build. Technically, we can change it to something else, but I think leaving as it is, makes it more explicit.